ember-file-upload/package.json

```diff
@@ -59,7 +59,7 @@
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.7.1 || ^6.0.1",
     "ember-cli-typescript": "^5.0.0",
-    "ember-modifier": "^2.1.2 || ^3.0.0",
+    "ember-modifier": "^3.0.0",
```

Comment:
Probably safer to actually use `^3.2.7` here.
The new API's were only added in `3.2.0`:
https://github.com/ember-modifier/ember-modifier/blob/master/CHANGELOG.md#v320-2022-03-29